### PR TITLE
Add support for module.strictExportPresence and output.futureEmitAssets

### LINKED ISSUE #205: Add futureEmitAssets to output
`config.output.futureEmitAssets(boolean)` is missing

### PR DESCRIPTION
**Pull request by yordis**
fixes #205 #206

### CODE REVIEW

**Comment by yordis** [comment]
@edmorley @eliperelman if you merge this, would you mind releasing a new version?

**Comment by yordis** [comment]
@edmorley I am sorry for bothering you, but could you give me a hand with this?

File: src/Output.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ module.exports = class extends ChainedMap {
       'strictModuleExceptionHandling',
       'umdNamedDefine',
       'webassemblyModuleFilename',
+      'futureEmitAssets',
```
Comment (edmorley) [inline src/Output.js]: Could you add this new entry after `'filename'`, to maintain alphabetical order?

**Comment by yordis** [comment]
@edmorley done

**Comment by edmorley** [comment]
Published in [6.2.0](https://www.npmjs.com/package/webpack-chain/v/6.2.0) :-)

**Pull request merged**